db/migrate/20150114141626_actually_rename_smart_proxy_auth_related_settings.rb

```diff
@@ -0,0 +1,16 @@
+class ActuallyRenameSmartProxyAuthRelatedSettings < ActiveRecord::Migration
+  def up
+    old_restrict_registered = Setting[:restrict_registered_puppetmasters]
+    execute "DELETE FROM settings WHERE name='restrict_registered_puppetmasters'"
+    Setting[:restrict_registered_smart_proxies] = old_restrict_registered unless old_restrict_registered.blank?
```

Comment:
`old_restrict_registered` may contain the *default* value of the setting (as `Setting[]` will return default if no value is set), so it'd be better to do `Setting.find_by_name` and set the value of smart_proxies if the value is present.